config/defaults.go

```diff
@@ -26,7 +26,7 @@ func DefaultConfig(home string) *NodeConfig {
 			MaxIdleTime:      3600 * time.Second,
 			MaxProofTime:     100 * time.Second,
 			BatchSubmitTime:  3600 * time.Second,
-			BatchSkew:        10,
+			BatchSkew:        24 * 7 * time.Hour,
```

Comment:
this is quite a lot for default no?

Reply:
yes, but by default shouldnt stop early.